docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,237 @@
+- Access to a running Teleport cluster.
+- A Linux server to run the Teleport Windows Desktop Service.
+  You can reuse an existing server running any other Teleport instance. 
+  However, you must generate a new invitation token to add the Windows Desktop Service
+  to an existing Teleport instance.
+- A physical or virtual machine running Microsoft Windows with Remote Desktop enabled 
+  and the RDP port—typically, port 3389—open for inbound connections from the Linux server.
 - (!docs/pages/includes/tctl.mdx!)
 
 ## Step 1/4. Prepare Windows
 
-In this section we'll import the Teleport certificate authority (**CA**) file
-to your Windows system, and prepare it for passwordless access through Teleport.
+Before you can enroll a Microsoft Windows desktop as a resource to be protected by Teleport,
+you need to prepare the Microsoft Windows environment with a certificate from the Teleport 
+certificate authority (CA) and modify the default authentication service for remote desktop
+access.
 
-### Import the Teleport root certificate
+To prepare a Windows computer:
 
-Export the Teleport user certificate authority by running the following from
-`cmd.exe` on your Windows system:
+1. Open a Command Prompt (`cmd.exe`) window on the Windows computer you want to enroll.
 
-```code
-$ curl -o teleport.cer https://teleport-proxy.example.com/webapi/auth/export?type=windows
-```
+1. Export the Teleport user certificate authority by running the following command:
+
+   ```code
+   $ curl -o teleport.cer https://teleport-proxy.example.com/webapi/auth/export?type=windows
+   ```
 
-### Install the Teleport service for Windows
+1. Download the Teleport Windows Auth [setup program](https://cdn.teleport.dev/teleport-windows-auth-setup-v(=teleport.version=)-amd64.exe).
 
-From the Windows system, download the [Teleport Windows Auth
-Setup](https://cdn.teleport.dev/teleport-windows-auth-setup-v(=teleport.version=)-amd64.exe).
-Run the installer. When prompted, select the Teleport certificate file from the previous step.
-Once complete, reboot the system.
+1. Doubleclick the executable you downloaded to run the Teleport Windows Auth Setup program 
+interactively and select the Teleport certificate that you exported when prompted.
 
-<Admonition type="tip" title="Headless Installation">
-The Teleport Windows Auth Setup can be run in a shell environment with elevated
-privileges. You can use this to automate installation, uninstallation, and
-certificate updates. The following command will install the Teleport Certificate,
-load the required DLL, disable NLA, and reboot the Windows machine:
+   The setup program:
+
+   - Installs the selected Teleport certificate.
+   - Loads the required dynamic link library (DLL) for Teleport to use.
+   - Disables Network Level Authentication (NLA) for remote desktop services. 
+
+1. Restart the computer.
+
+
+After you download the setup program so it's available on the Windows computer, 
```

Comment:
I don't think we need to say 'after you downloaded the program' - we already told the user to download it and double click it.

We probably just want to say something like "If you prefer an automated install, you can invoke the installer from an administrative command prompt"